src/BugsnagUnity/ErrorTypes.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+namespace BugsnagUnity
+{
+    public class ErrorTypes
```

Comment:
More of a general question but do you feel that the usage of `ErrorTypes` is idiomatic (for C# in general and Unity specifically)? The combination of setting various fields to `true` and/or using `SetAllDisabled()` seems a bit convoluted for a property on configuration though I haven't seen a lot of similar implementations. Would a flag enum be too low level?